CSLabs.Api/Proxmox/ProxmoxDBApi.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Threading.Tasks;
+using CSLabs.Api.Models;
+using CSLabs.Api.Models.HypervisorModels;
+
+namespace CSLabs.Api.Proxmox
+{
+    public class ProxmoxDBApi : ProxmoxApi
```

Comment:
You havent configured this ProxmoxDBApito be used anywhere but one place, it should be used instead of the Proxmox api since you want to track the start and stop states of the VM when initiated by the UI.